apis/storage/v1alpha3/bucket_types.go

```diff
@@ -141,30 +133,6 @@ type S3BucketClassList struct {
 	Items           []S3BucketClass `json:"items"`
 }
 
-// GetBucketName based on the NameFormat spec value,
-// If name format is not provided, bucket name defaults to UID
-// If name format provided with '%s' value, bucket name will result in formatted string + UID,
-//   NOTE: only single %s substitution is supported
-// If name format does not contain '%s' substitution, i.e. a constant string, the
-// constant string value is returned back
-//
-// Examples:
-//   For all examples assume "UID" = "test-uid"
-//   1. NameFormat = "", BucketName = "test-uid"
-//   2. NameFormat = "%s", BucketName = "test-uid"
-//   3. NameFormat = "foo", BucketName = "foo"
-//   4. NameFormat = "foo-%s", BucketName = "foo-test-uid"
-//   5. NameFormat = "foo-%s-bar-%s", BucketName = "foo-test-uid-bar-%!s(MISSING)"
-func (b *S3Bucket) GetBucketName() string {
-	if b.Spec.NameFormat == "" {
-		return string(b.GetUID())
-	}
-	if strings.Contains(b.Spec.NameFormat, "%s") {
-		return fmt.Sprintf(b.Spec.NameFormat, string(b.GetUID()))
-	}
-	return b.Spec.NameFormat
-}
```

Comment:
I always found this format-string-with-uid pattern a bit strange. I'm glad the last examples are finally being removed. :)